tardis/montecarlo/base.py

```diff
@@ -39,6 +40,8 @@ class MontecarloTransport(HDFWriterMixin):
     """
     This class is designed as an interface between the Python part and the
     montecarlo C-part
+
+    note:: TARDIS must be built with OpenMP support in order for ``nthreads`` to have effect.
```

Comment:
That's not true anymore so it can be removed, `nthreads` is just used to limit the numba thread count

Reply:
Removed this.